cves/kernel/CVE-2012-6657.yml

```
@@ -147,10 +142,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: Discovered by a developer, no used of tools were mentioned.
```

Small rewording change could be made here for clarification 
```suggestion
  answer: Discovered by a developer and no use of automated tools was mentioned.
```